### PR TITLE
Removed stop interface from profilers

### PR DESCRIPTION
**Pull request by akaStiX**
Removed useless stop interface from profilers - it was doing nothing. Keep it simple :)

### CODE REVIEW

File: vendor/arm/mali/mali_profiler.cpp [hunk 1]
```diff
@@ -363,6 +363,14 @@ MaliProfiler::MaliProfiler(const GpuCounterSet &enabled_counters) :
 	}
 }
 
+MaliProfiler::~MaliProfiler()
```
Comment (TomAtkinsonArm) [inline vendor/arm/mali/mali_profiler.cpp:366]: Can this be removed as it is [implemented here](https://github.com/ARM-software/HWCPipe/pull/34/files#diff-ca3abba14a5fc4c875fcdecf08e1ec0bR58)?

File: vendor/arm/mali/mali_profiler.h [hunk 2]
```diff
@@ -38,7 +38,7 @@ class MaliProfiler : public GpuProfiler
 {
   public:
 	explicit MaliProfiler(const GpuCounterSet &enabled_counters);
-	virtual ~MaliProfiler() = default;
+	virtual ~MaliProfiler();
```
Comment (TomAtkinsonArm) [inline vendor/arm/mali/mali_profiler.h:41]: Again this wont need to be changed. See above comment

**Review by TomAtkinsonArm — COMMENTED** [review]
LGTM. Some changes are implemented in your next PR

File: vendor/arm/mali/mali_profiler.cpp (hunk 1 above)
Comment (akaStiX) [reply vendor/arm/mali/mali_profiler.cpp:366]: Yes, if we'll come up with a conclusion on what do you want to pull-in I can clean up the rest

Comment (TomAtkinsonArm) [reply vendor/arm/mali/mali_profiler.cpp:366]: Lets focus this PR on removing the stop interface only

**Pull request closed without merging**